services/transactionpool/service.go

```diff
@@ -33,8 +33,11 @@ func (s *service) AddNewTransaction(input *services.AddNewTransactionInput) (*se
 			SignedTransactions: []*protocol.SignedTransaction{input.SignedTransaction},
 		},
 	})
+
 	//This is commented out because currently transport broadcast will also broadcast to myself. So HandleForwardedTransactions will be the on to add this transaction.
```

Comment:
LOL, you removed the commented code but not the comment. This is why comments not inline with the commented code are evil